llama_hub/library.json

```diff
@@ -540,5 +540,15 @@
   "KibelaReader": {
     "id": "kibela",
     "author": "higebu"
+  },
+  "GitHubIssuesReader": {
```

Comment:
Sorry should've clarified, this needs to match the class name: GitHubRepositoryIssuesReader

Reply:
Thanks for clarification, fixed!